assets/sass/components/sign-in-with-google/_googlesitekit-sign-in-with-google-settings-module.scss

```diff
@@ -17,9 +17,7 @@
  */
 
 .googlesitekit-plugin {
-
```

Comment:
We do commonly have an empty line after a class selector declaration, there's no need for the whitespace changes in this PR. Please can you revert them, to reduce the "line noise" :)